README.md

```diff
@@ -1,68 +1,159 @@
+Start with the [GitHub Wiki](https://github.com/VilnaCRM-Org/infrastructure-template/wiki) for deeper guides and troubleshooting tips. Community Q&A lives under the [`vilnacrm` tag on Stack Overflow](https://stackoverflow.com/questions/tagged/vilnacrm).
+
+## Testing and Validation
 
-## Tests
-[Test status](https://github.com/VilnaCRM-Org/infrastructure-template/actions)
+Continuous integration runs automatically on every pull request. You can also validate locally:
 
-If this isn't passing, is there something you can do to help?
+- `make pulumi-preview` to review planned resources.
+- `make pulumi-up` followed by `pulumi stack output` to inspect applied results.
+- `pulumi stack history` (from inside the container) to audit state transitions.
+
+## Repository Synchronization
+
+This template feeds other VilnaCRM infrastructure projects through [`actions-template-sync`](https://github.com/AndreasAugustin/actions-template-sync). For authentication guidance, see:
+
+- `.github/TEMPLATE_SYNC_PAT.md` for Personal Access Tokens.
+- `.github/TEMPLATE_SYNC_APP.md` for GitHub App credentials.
+
+Remember to grant least privilege, rotate secrets regularly, and monitor workflow logs.
 
 ## Security
-Please disclose any vulnerabilities found responsibly – report security issues to the maintainers privately.
 
-See [SECURITY](https://github.com/VilnaCRM-Org/infrastructure-template/tree/main/SECURITY.md) and [Security advisories on GitHub](https://github.com/VilnaCRM-Org/infrastructure-template/security).
+Report security issues responsibly to the maintainers. Consult [`SECURITY.md`](SECURITY.md) and the repository’s [security advisories](https://github.com/VilnaCRM-Org/infrastructure-template/security).
 
 ## Contributing
-Please submit bug reports, suggestions, and pull requests to the [GitHub issue tracker](https://github.com/VilnaCRM-Org/infrastructure-template/issues).
-
-We're particularly interested in fixing edge cases, expanding test coverage, and updating translations.
 
-If you found a mistake in the docs, or want to add something, go ahead and amend the wiki – anyone can edit it.
+Bug reports, feature requests, and pull requests are welcome via the [issue tracker](https://github.com/VilnaCRM-Org/infrastructure-template/issues). Documentation updates are highly appreciated—feel free to enhance the wiki or submit fixes directly.
 
 ## Sponsorship
+
 Development time and resources for this repository are provided by [VilnaCRM](https://vilnacrm.com/), the free and opensource CRM system.
 
 Donations are very welcome, whether in beer 🍺, T-shirts 👕, or cold, hard cash 💰. Sponsorship through GitHub is a simple and convenient way to say "thank you" to maintainers and contributors – just click the "Sponsor" button [on the project page](https://github.com/VilnaCRM-Org/infrastructure-template). If your company uses this template, consider taking part in the VilnaCRM's enterprise support program.
+
```

Comment:
We should remove the terraform docs and all terraform related stuff from the infrastructure template